src/benchmarkstt/input/core.py

```diff
@@ -8,6 +8,9 @@
 
 
 class PlainText(input.Base):
+    """
+    Plain text
```

Comment:
When in the context of `benchmarkstt --help`, the difference between `argument` and `plaintext` a little confusing.

```
reference and hypothesis types:
  You can specify which file type the --reference/-r and --hypothesis/-h
  arguments should be treated as.

  Available types:
   - 'infer': Automatically infers type from the filename extension
   - 'argument': Read the argument and treat as plain text (without reading
  from file)
   - 'plaintext': Plain text
```

Maybe something like "Read the argument as a name for a plain text file" instead of just "Plain text"

Reply:
Suggestion: “treat file as plain text” seems sufficient? Don’t like adding in ‘argument’ as the docs would be applicable to more than the cli tool (eg. Api, straight python code usage)

Reply:
Changed explanation to:

>   Available types:
>   'infer': Load from a given filename. Automatically infer file type from
>   the filename extension.
>   'argument': Read the argument and treat as plain text (without reading
>   from file)
>   'plaintext': Load from a given filename. Treat file as plain text

This seems adequate?

Reply:
That's great!